openfisca_us/variables/gov/states/in/tax/income/deductions/in_count_children_nonpublic_school.py

```diff
@@ -0,0 +1,10 @@
+from openfisca_us.model_api import *
+
+
+class in_count_children_nonpublic_school(Variable):
```

Comment:
Can you change this to person-level and move it to https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/variables/household/demographic/person as `is_in_k12_nonpublic_school`, a la the existing `is_in_k12_school`? Similar to rent, making this US-wide will save space and serve similar state laws, at the cost of some precision.